src/pages/Likes.js

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from "react";
```

Comment:
changes: I cannot see removing liked video cards from the likes page. I think it should be present.

Reply:
as per user stories in lms, likes page or option to remove a liked video from likes page isn't required. So, i'll skip it for now.